couper.hcl

```
@@ -10,11 +10,12 @@ server "oidc-gate" {
         authorization = request.headers.authorization
       }
       backend {
-        origin = env.ORIGIN
-        hostname = env.ORIGIN_HOSTNAME != "" ? env.ORIGIN_HOSTNAME : request.host
-        connect_timeout = env.CONNECT_TIMEOUT
-        ttfb_timeout = env.TTFB_TIMEOUT
-        timeout = env.TIMEOUT
+        origin = env.BACKEND_ORIGIN
+        hostname = env.BACKEND_HOSTNAME != "" ? env.BACKEND_HOSTNAME : request.host
+        connect_timeout = env.BACKEND_CONNECT_TIMEOUT
+        ttfb_timeout = env.BACKEND_TTFB_TIMEOUT
+        timeout = env.BACKEND_TIMEOUT
+        disable_certificate_validation = env.BACKEND_DISABLE_CERTIFICATE_VALIDATION
```

The environment string mapping from string `true` or `false` to a boolean one is supported for the `settings` block only. So in this case the assignment must be slightly changed.
```suggestion
        disable_certificate_validation = env.BACKEND_DISABLE_CERTIFICATE_VALIDATION == "true"
```